lib/internal/modules/cjs/loader.js

```diff
@@ -1158,13 +1159,19 @@ function wrapSafe(filename, content, cjsModuleInstance) {
       '__dirname',
     ], {
       filename,
+      cachedData: codeCache,
       importModuleDynamically(specifier, _, importAssertions) {
         const cascadedLoader = getCascadedLoader();
         return cascadedLoader.import(specifier, normalizeReferrerURL(filename),
                                      importAssertions);
       },
     });
 
+    // The code cache is used for SEAs only.
+    if (codeCache && result.cachedDataRejected !== false && isSea()) {
```

Comment:
I think if we just do

```suggestion
    if (codeCache && result.cachedDataRejected !== false && internalBinding('sea').isSea()) {
```

we would not need to load the sea binding in normal non-SEA code paths (and therefore do not have to update test-bootstrap-modules.js). Even in an SEA this is only supposed to be load once anyway.